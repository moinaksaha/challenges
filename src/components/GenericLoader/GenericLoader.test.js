import React from 'react'
import { shallow } from 'enzyme';
import Genericloader from './index.js'
import toJson from 'enzyme-to-json';

describe('Component: <Genericloader />', () => {

  test('Render Status: Success', () => {
    const wrapper = shallow(<Genericloader />);
    expect(wrapper.exists()).toBe(true);
  })

  test('Snapshot: Enzyme', () => {
    const tree = shallow(<Genericloader />)
    expect(toJson(tree)).toMatchSnapshot()
  })
  
})