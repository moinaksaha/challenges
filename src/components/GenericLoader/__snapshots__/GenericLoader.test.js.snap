// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Genericloader /> Snapshot: Enzyme 1`] = `
<styled.div>
  <img
    height={30}
    src="loader.gif"
    width={30}
  />
  <p>
    Loading, please wait ...
  </p>
</styled.div>
`;
